extensions/adobe/experience/decisioning/offerItem.schema.json

```diff
@@ -0,0 +1,123 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/offeritem",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Offer Item",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "meta:status": "experimental",
+  "description": "Class which represents an Offer",
+  "definitions": {
+    "offeritem": {
+      "properties": {
+        "xdm:lifecycleStatus": {
+          "title": "life cycle status",
+          "type": "string",
+          "description": "Status associated with offer life cycle.",
+          "meta:enum": {
```

Comment:
If these are the expected values, please add the "enum" too.

Reply:
ok, here the enum values can change at a later point. hence soft enum and not backing with array..